htpasswd.go

```diff
@@ -100,7 +100,8 @@ func (h *HtpasswdFile) throttleTimer() {
 	}
 }
 
-var validUsernameRegexp = regexp.MustCompile(`^[\p{L}@.-]+$`)
+// FIXME: Use folderPathValid() instead once PR #112 is merged
```

Comment:
I'm not sure I understand that comment. Is the idea that the folder path splitting will prevent usernames containing slashes from working anyways?

Reply:
No, it's about having one place to validate this. PR #112 introduces a function that can be used to check this, since path escaping is the actual issue.

I have not actually looked in depth into the original issue that PR #132 resolves. I suspect that PR #112 actually resolves the underlying issue that PR #132 is trying to work around, so perhaps this whole check is not even needed once PR #112 is merged.

Reply:
If I remember the code in #112 correctly then it'll resolve the underlying issue. Can we do without the comment for now? It'll look odd in the history...

Reply:
Removed the comment and squashed the commits.